editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/features/misc/ReconnectNodesFeature.java

```diff
@@ -51,5 +63,44 @@ public void postReconnect(IReconnectionContext context) {
 			oldTarget.setOutputElement(null);
 		}
 		CamelElementConnection f = new CamelElementConnection(source, newTarget);
```

Comment:
the f variable is not used afterwise, even if this code has not been modified, it seems strange, do we need to call the constructor at all? At least we should not assign it to avairable